Updated Security Model and FAQ pages (#7528)
hashicorp/consul: website/source/docs/faq.html.md

situations where a limited subset of data can be replicated, such as with Consul's built-in
[ACL replication](https://learn.hashicorp.com/consul/day-2-operations/acl-replication) capability, or
external tools like [consul-replicate](https://github.com/hashicorp/consul-replicate).

## Q: Can Consul natively handle protecting against other processes accessing Consul's memory state? 

> In short, Consul does not have anything built in to protect against memory 

Reviewer: ```suggestion
Consul does not provide built-in memory access protections, and doesn't interact with the host system to change or manipulate 
```

> viewing doesn't interact with the host system to change or manipulate 

Reviewer 2: missing an "and" here?

application security.

> As Consul is a singular binary, we recommend taking any precautions or 

Reviewer: ```suggestion
We recommend taking any precautions or 
```

remediation steps that you would normally do for individual processes, based 
on your operating system. 

Please see our 

> [Security Model](https://www.consul.io/docs/internals/security.html) for more.

Reviewer: ```suggestion
[Security Model](https://www.consul.io/docs/internals/security.html) for more information.
```
